notebooks/official/workbench/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -1,16 +1,68 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    if delete_bucket or os.getenv("IS_TESTING"):
</pre><p>You can simplify to <code>if delete_bucket</code>. We have cleanup scripts for the CI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>